libshaderc_util/include/libshaderc_util/compiler.h

```diff
@@ -103,11 +106,20 @@ class Compiler {
 
   // Target environment.
   enum class TargetEnv {
-    Vulkan,
-    OpenGL,
+    Vulkan,  // Default to Vuilkan 1.0
```

Comment:
Vuilkan

Reply:
oh my.  I must have been tired.  Actually I was on a plane and comments were dark blue on black... :-(